### PR TITLE
Add helper to add layouts

### PR DESCRIPTION
**Pull request by kaelig**
See http://sassmeister.com/gist/ab2e1ce4cbd4b5b7efa4 for a demo

### CODE REVIEW

**Comment by kaelig** [comment]
This is to circumvent the issue that @wheresrhys had when trying to add custom layouts in Next.

Closes https://github.com/Financial-Times/o-grid/pull/74 and should fix https://github.com/Financial-Times/next-sass-setup/pull/22#discussion_r26352114

File: src/scss/_main.scss [hunk 1]
```diff
@@ -3,6 +3,41 @@
 /// @link http://registry.origami.ft.com/components/o-grid
 ////
 
+/// Add a layout
+///
+/// @example scss
+///  @include oGridAddLayout($layout-name: XS, $colum-width: 20px);
+///  @include oGridAddLayout($layout-name: P, $viewport-width: 500px);
```
Comment (AlbertoElias) [inline src/scss/_main.scss]: viewport-width?

File: src/scss/_main.scss [hunk 2]
```diff
@@ -3,6 +3,41 @@
 /// @link http://registry.origami.ft.com/components/o-grid
 ////
 
+/// Add a layout
+///
+/// @example scss
+///  @include oGridAddLayout($layout-name: XS, $colum-width: 20px);
```
Comment (AlbertoElias) [inline src/scss/_main.scss]: typo in column

**Comment by AlbertoElias** [comment]
lgtm

**Comment by kaelig** [comment]
Cheers, good spots!

**Pull request merged**